.github/workflows/changelog_generation.yml

```
@@ -0,0 +1,56 @@
+name: Compile changelogs
+
+on:
+  schedule:
+  - cron: "0 0 * * *"
```

```suggestion
on:
  push:
    branches:
    - master
```
Or say why you need to schedule it by cron

---

Ладно, зря быканул

---

> master
ok